openshift-knative-operator/hack/004-rbac-proxy.patch

```diff
@@ -1,9 +1,37 @@
 diff --git a/openshift-knative-operator/cmd/operator/kodata/knative-serving/0.19.0/0-rbac-proxy.yaml b/openshift-knative-operator/cmd/operator/kodata/knative-serving/0.19.0/0-rbac-proxy.yaml
 new file mode 100644
-index 00000000..37558cd9
+index 00000000..d9df9652
 --- /dev/null
 +++ b/openshift-knative-operator/cmd/operator/kodata/knative-serving/0.19.0/0-rbac-proxy.yaml
-@@ -0,0 +1,314 @@
+@@ -0,0 +1,342 @@
++apiVersion: rbac.authorization.k8s.io/v1
++kind: Role
++metadata:
++  name: knative-serving-prometheus-k8s
++rules:
++  - apiGroups:
++      - ""
++    resources:
++      - services
++      - endpoints
++      - pods
++    verbs:
++      - get
++      - list
++      - watch
++---
++apiVersion: rbac.authorization.k8s.io/v1
++kind: RoleBinding
++metadata:
++  name: knative-serving-prometheus-k8s
++roleRef:
++  apiGroup: rbac.authorization.k8s.io
++  kind: Role
++  name: knative-serving-prometheus-k8s
++subjects:
++  - kind: ServiceAccount
++    name: prometheus-k8s
++    namespace: openshift-monitoring
```

Comment:
The part bellow will be removed (most likely) because prometheus-k8s account already has the required role:
```
$ oc describe clusterroles prometheus-k8s
Name:         prometheus-k8s
Labels:       <none>
Annotations:  <none>
PolicyRule:
  Resources                                  Non-Resource URLs  Resource Names  Verbs
  ---------                                  -----------------  --------------  -----
  tokenreviews.authentication.k8s.io         []                 []              [create]
  subjectaccessreviews.authorization.k8s.io  []                 []              [create]
                                             [/metrics]         []              [get]
  namespaces                                 []                 []              [get]
  nodes/metrics                              []                 []              [get]
```

Reply:
yes prometheus-k8s SA already has the required permissions